platform-sdk/docs/core/platform-status.md

```diff
@@ -0,0 +1,65 @@
+# Platform Status
+
+The platform status is represented as an enum value from `PlatformStatus.java`. It is updated via
+`SwirldsPlatform::checkPlatformStatus`, which is passed around as a callback to any object that has the ability to
+effect a change in platform status.
+
+A diagram of expected state transitions is available [here](./platform-status-transitions.svg). Note that there are
+currently no hard restrictions on state transitions: this diagram is merely a representation of the expected behavior.
+
+## Used Statuses
+
+The following statuses, along with their respective triggers, are listed in order of precedence (from highest to lowest):
+
+- `DISCONNECTED`: There is more than one node in the network, and the number of active connections is zero
+- `BEHIND`: The sync manager has reported that the node has fallen behind
+- `MAINTENANCE`: The freeze manager has reported that a freeze has started
+- `FREEZE_COMPLETE`: The freeze manager has reported that a freeze has completed
+- `ACTIVE`: None of the previously defined statuses apply
+
+## Detailed Trigger Explanations
+
+### `DISCONNECTED`
+
+- `SwirldsPlatform::checkPlatformStatus` is called in `SwirldsPlatform::newConnectionOpened` and
+`SwirldsPlatform::connectionClosed`, which are called each time a new connection is opened or closed, respectively
+- If the number of active connections becomes zero, or ceases to be zero, the status is updated to `DISCONNECTED`, or
+falls through to the status with next highest precedent
+
+### `BEHIND`
+
+- Each time the `ShadowGraphSynchronizer` syncs with another node, it checks whether it has fallen behind that other node
+- If it has fallen behind, then `FallenBehindManagerImpl::reportFallenBehind` is called
+- If this call causes the total number of fallen behind reports to exceed the threshold, then
+  `SwirldsPlatform::checkPlatformStatus` is called, which results in the status transitioning to `PlatformStatus::BEHIND`
+  - This only occurs when the threshold is first crossed, and not on subsequent reports
```

Comment:
The only way to get out of a this status is to perform a reconnect.

Reply:
Added